src/Makefile.am

```diff
@@ -40,6 +40,7 @@ noinst_HEADERS = \
 	app-layer-nfs-udp.h \
 	app-layer-ntp.h \
 	app-layer-parser.h \
+    app-layer-pgsql.h \
```

Comment:
formatting looks off

Reply:
Thank you, and I'm sorry for having to have that one pointed out >_<' . In VSCode they looked the same...